lib/active_force/sobject.rb

```diff
@@ -7,7 +7,7 @@
 require 'forwardable'
 require 'logger'
 require 'restforce'
-require 'active_model/uninitialized_sobject.rb'
+require 'active_model/uninitialized_value.rb'
```

Comment:
The `.rb` shouldn't be necessary here and it's not common that it's included.  Would you mind chopping it off, please?